docs/archive-data.md

```diff
@@ -136,58 +100,60 @@ namespaceDefaults:
+| `archival.history.provider`                    | Sub provider configs are `filestore`, `gstorage`, `s3`, or `your_custom_provider`. |  The default config specifies `filestore`. |
+| `archival.history.provider.filestore.fileMode` | File permission string | This specifies the file permissions of the archived files. We recommend using the default value of `"0666"` to avoid read/write issues. |
+| `archival.history.provider.filestore.dirMode`  | File permission string | This specifies the directory permissions of the archive directory. we recommend using the default value of `"0766"` to avoid read/write issues. |
+| `namespaceDefaults.archival.history.state`     | `enabled`, `disabled`  | This sets the default state of the Archival feature whenever a new Namespace is created without specifying the Archival state. |
+| `namespaceDefaults.archival.history.URI`       | Valid URI              | The value must be a URI of the file store location and match a schema that correlates to a provider. |
 
-At the cluster level, `archival.history.state` must be set to `"enabled"` for Archival to work at the Namespace level for Even Histories. Once enabled at the cluster level, a Namespace can safely switch between `enabled` and `disabled` states.
-
-Archival is supported in global Namespaces (Namespaces spanning multiple clusters). When Archival is running in a global Namepsace, it will first run on the active cluster and then some time later it will run on the standby cluster. Before archiving, a history check is done to see what has been previously uploaded.
+### Namespace creation
 
-## Archive reads
- 
-To retrieve archived Event Histories you will need the Workflow Id and the Run Id:
+While Archival is configured at the cluster level, it operates independently within each Namespace. When a Namespace is created, if an Archival URI is not specified then the Namespace will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file. The Archival URI can not be changed once the Namespace is created. Each Namespace supports a single Archival URI, but each Namespace can use a different URI (archiver). A Namespace can safely switch Archival between `enabled` and `disabled` states as long as Archival is enabled at the cluster level.
 
-```bash
-$ ./temporal --ns <namespace> wf show --wid <workflowId> --rid <runId>
-```
+Archival is supported in global Namespaces (Namespaces spanning multiple clusters). When Archival is running in a global Namepsace, it will first run on the active cluster and then some time later it will run on the standby cluster. Before archiving, a history check is done to see what has been previously archived.
 
-:::tip
+### Test setup
 
-Want to test this feature locally? Start by running a Temporal server:
+To test Archival locally, start by running a Temporal server:
 
 ```bash
-$ ./temporal-server start
+./temporal-server start
```

Comment:
We should reconsider including this here as we have to maintain it and most people use docker

Reply:
docker-compose doesn't support Archival at present - so this would be the only way to test (though whether this would actually work is unknown - but it was pulled from existing docs).